capnslog/formatters.go

```diff
@@ -28,6 +29,28 @@ type Formatter interface {
 	Flush()
 }
 
+func NewSimpleFormatter(w io.Writer) *SimpleFormatter {
+	return &SimpleFormatter{
+		w: bufio.NewWriter(w),
+	}
+}
+
+type SimpleFormatter struct {
```

Comment:
I know we've failed at this so far, but docstrings explaining the different formatters would be very helpful for consumers